USERS-GUIDE.md

```diff
@@ -579,7 +579,7 @@ used simultaneously.
 ### XML Schema-based Configuration
 
 _conf4j_ provides custom configuration schema `http://www.sabre.com/schema/oss/conf4j` which exposes three custom tags:
-`<conf4j:configure/>`, `<conf4j:configuration/>` and `<conf4j:configuration-scan/>`.
+`<conf4j:configure/>`, `<conf4j:configuration/>`, `<conf4j:converter/>`, `<conf4j:delegatingConverterFactory/>` and `<conf4j:configuration-scan/>`.
```

Comment:
`delegatingConverterFactory` name is wrong - it should be `converter-decorator`. Please fix in everywhere in the documentation.